### PR TITLE
9.0 hr  holidays compute days

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Update the README to the latest template (removing the issue part that you haven't updated): https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The README isn't yet fully updated (it last the issue part, which removes the link)

File: hr_holidays_compute_days/README.rst [hunk 1]
```diff
@@ -24,15 +24,16 @@ Usage
...
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/hr/issues/new?body=module:%20hr_holidays_compute_days%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/hr/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+`here <https://github.com/OCA/hr/issues/new?body=module:%20hr_holidays_computed_days%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (feketemihai) [inline hr_holidays_compute_days/README.rst:36]: @rami-wafaie Thanks for porting the module, please change version in the issues, from 8 to 9.

**Review by feketemihai — CHANGES_REQUESTED** [review]
@rami-wafaie Since the module is already ported to version 10 by @yelizariev , can i ask you to spend a little time and to improve the module just like in v10, it's about an onchange method of holiday_status_id, please see the explanations from https://github.com/OCA/hr/pull/261, and the method it's over here: https://github.com/OCA/hr/blob/10.0/hr_holidays_compute_days/models/hr_holidays.py#L28.

Thanks again for your work.

**Comment by gurneyalex** [comment]
@rami-wafaie hello, I'm in a quite urgent need of this module. Do you need help for making the requested changes?

**Comment by gurneyalex** [comment]
rebasing your pr on top of 9.0 will ensure that #263 is present and avoid a red build on runbot.

**Comment by rami-wafaie** [comment]
@gurneyalex Hello, The needed changes require some time (I already spent time on it last week but still no solution), and this week Im quite busy, if you can help me with the needed changes from @feketemihai I will be thankful

**Comment by feketemihai** [comment]
superseeded by #270. closing this one...

**Pull request closed without merging**